components/test_raftstore/src/cluster.rs

```diff
@@ -395,7 +395,7 @@ impl<T: Simulator> Cluster<T> {
         let region_id = request.get_header().get_region_id();
         loop {
             let leader = match self.leader_of_region(region_id) {
-                None => return Err(box_err!("can't get leader of region {}", region_id)),
+                None => return Err(Error::NotLeader(region_id, None)),
```

Comment:
But `leader_of_region` already tries for 5 seconds.

Reply:
I think the timeout should be controlled by `request`.